src/pybind/mgr/dashboard/frontend/src/app/shared/enum/fixed-color-class.enum.ts

```diff
@@ -0,0 +1,7 @@
+export enum FixedColorClass {
+  mds = '0',
+  mgr = '1',
+  mon = '2',
+  osd = '3',
+  rgw = '4'
+}
```

Comment:
My preference is that components should be open and flexible rather than opinionated and focused on specific cases. So we should create generic components that can be simply customized (but from the outside of the component not the inside).